lib/LinkEmbedder/Link/Basic.pm

```diff
@@ -14,6 +14,10 @@ sub learn_p {
 
   $self->type($type);
 
+  if ($type eq 'video' and $url =~ m/\.([^.]+)$/) {
+    $self->mimetype("video/$1");
```

Comment:
Maybe do `mimetype(lc "video/$1")` in case the extension is .WEBM, instead of .webm.

Reply:
Ok